services/slam/server_test.go

```diff
@@ -62,7 +62,7 @@ func TestServer(t *testing.T) {
 	})
 
 	t.Run("working get map function", func(t *testing.T) {
-		pose := spatial.NewPose(r3.Vector{1, 2, 3}, &spatial.OrientationVector{math.Pi / 2, 0, 0, -1})
+		pose := spatial.NewPose(r3.Vector{X: 1, Y: 2, Z: 3}, &spatial.OrientationVector{Theta: math.Pi / 2, OX: 0, OY: 0, OZ: -1})
```

Comment:
why does this need to change?

Reply:
Formatting, my VScode has a checker to tell you when struct variables are not be referenced but instead placed in a certain order. This is more verbose and accurate